.github/workflows-src/partials/docs.yml

```diff
@@ -21,8 +21,8 @@ docs:
     - name: Upload docs
       run: node .github/upload-preview.js packages/react-docs/public
       if: always()
-#    - name: a11y tests
-#      run: yarn serve:docs & yarn test:a11y
-#    - name: Upload a11y results
-#      run: node .github/upload-preview.js packages/react-docs/coverage
-#      if: always()
+    - name: a11y tests
+      run: yarn serve:docs & yarn test:a11y
+    - name: Upload a11y results
+      run: node .github/upload-preview.js packages/react-docs/coverage
```

Comment:
In `upload-preview.js`, this conditional adds the a11y report comment https://github.com/patternfly/patternfly-react/blob/9de508fa1e8e0588c389c29c4d478f547544856a/.github/upload-preview.js#L72-L74

It tests for `uploadFolderName === 'dist'`, where `uploadFolderName` is the last string in the argument passed (`packages/react-docs/coverage`). So we need to update the conditional in `upload-preview.js` to test for "coverage" instead of "dist".